pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -213,7 +214,23 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
 				authorizationapi.NewRule("create").Groups(buildGroup).Resources(authorizationapi.JenkinsPipelineBuildResource).RuleOrDie(),
 			},
 		},
-
+		{
+			ObjectMeta: kapi.ObjectMeta{
+				Name: StorageAdminRoleName,
+			},
+			Rules: []authorizationapi.PolicyRule{
+				authorizationapi.NewRule(readWrite...).Groups(kapiGroup).Resources("bindings", "endpoints", "persistentvolumes", "persistentvolumes/status",
+					"secrets", "services").RuleOrDie(),
+				authorizationapi.NewRule(read...).Groups(kapiGroup).Resources("events", "limitranges", "namespaces", "namespaces/status",
+					"resourcequotas", "resourcequotas/status", "resourcequotausages").RuleOrDie(),
+				authorizationapi.NewRule(read...).Groups(projectGroup).Resources("projects").RuleOrDie(),
+				authorizationapi.NewRule(read...).Groups(kapiGroup).Resources("persistentvolumeclaims", "persistentvolumeclaims/status").RuleOrDie(),
+				authorizationapi.NewRule(read...).Groups(kapiGroup).Resources("securitycontextconstraints", "serviceaccounts").RuleOrDie(),
+				authorizationapi.NewRule(read...).Groups(kapiGroup).Resources("pods/binding", "pods/log", "pods/status").RuleOrDie(),
```

Comment:
same as above, but removed and agree, we can start as lean as possible and then extend when needed